roachtest: npgsql failed

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12963626?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12963626?buildTab=artifacts#/npgsql) on release-23.2 @ [8c838306785cc9cf0f8f7b9ad0ec052f3ede9787](https://github.com/cockroachdb/cockroach/commits/8c838306785cc9cf0f8f7b9ad0ec052f3ede9787):



```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-beta.2-dev-8c838306785cc9cf0f8f7b9ad0ec052f3ede9787
Tests run against npgsql v7.0.2
3280 Total Tests Run
2580 tests passed
700 tests failed
248 tests skipped
61 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CopyTests(NonMultiplexing).Wrong_format_text_import -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_metamorphicBuild=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12963626/npgsql/1701689881473/1701690196395)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34082

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/13016974?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/13016974?buildTab=artifacts#/npgsql) on release-23.2 @ [004cb44f70db685052a50afe78bc2df566ad26f9](https://github.com/cockroachdb/cockroach/commits/004cb44f70db685052a50afe78bc2df566ad26f9):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-beta.2-dev-004cb44f70db685052a50afe78bc2df566ad26f9
Tests run against npgsql v7.0.2
3280 Total Tests Run
2577 tests passed
703 tests failed
248 tests skipped
61 tests ignored
0 tests passed unexpectedly
4 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).Cached_command_double_dispose -  (unexpected)
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).TwoNonQueries -  (unexpected)
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).CloseConnection -  (unexpected)
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).TwoQueries -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_metamorphicBuild=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-13016974/npgsql/1701949906114/1701950216424)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/13359425?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/13359425?buildTab=artifacts#/npgsql) on release-23.2 @ [663609a172f53aa0cd8114dc0f90e46af03cabc6](https://github.com/cockroachdb/cockroach/commits/663609a172f53aa0cd8114dc0f90e46af03cabc6):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-rc.2-dev-663609a172f53aa0cd8114dc0f90e46af03cabc6
Tests run against npgsql v7.0.2
3280 Total Tests Run
2580 tests passed
700 tests failed
248 tests skipped
61 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).NonQueryQuery -  (unexpected)
For a full summary look at the npgsql artifacts 
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```

Parameters:
 - <code>ROACHTEST_arch=amd64</code>
 - <code>ROACHTEST_cloud=gce</code>
 - <code>ROACHTEST_coverageBuild=false</code>
 - <code>ROACHTEST_cpu=4</code>
 - <code>ROACHTEST_encrypted=false</code>
 - <code>ROACHTEST_metamorphicBuild=false</code>
 - <code>ROACHTEST_ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-13359425/npgsql/1704196095386/1704196415747)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #117234 roachtest: npgsql failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.2.0-rc release-blocker]
- #117169 roachtest: npgsql failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-master]
- #116436 roachtest: npgsql failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-23.2.0-rc]
- #115863 roachtest: npgsql failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/hibernate_blocklist.go
+++ b/pkg/cmd/roachtest/tests/hibernate_blocklist.go
@@ -24,6 +24,7 @@ var hibernateSpatialIgnoreList = blocklist{
 }
 
 var hibernateIgnoreList = blocklist{
+	"org.hibernate.orm.test.batch.BatchTest.testBatchInsertUpdate":                                       "flaky",
 	"org.hibernate.serialization.SessionFactorySerializationTest.testUnNamedSessionFactorySerialization": "flaky",
 	"org.hibernate.serialization.SessionFactorySerializationTest.testNamedSessionFactorySerialization":   "flaky",
 	"org.hibernate.test.batch.BatchTest.testBatchInsertUpdate":                                           "flaky",

--- a/pkg/cmd/roachtest/tests/npgsql_blocklist.go
+++ b/pkg/cmd/roachtest/tests/npgsql_blocklist.go
@@ -676,7 +676,9 @@ var npgsqlBlocklist = blocklist{
 }
 
 var npgsqlIgnoreList = blocklist{
+	`Npgsql.Tests.CommandTests(Multiplexing).Command_Dispose_does_not_close_reader`:                   "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Cursor_move_RecordsAffected `:                            "flaky",
+	`Npgsql.Tests.CommandTests(Multiplexing).Multiple_statements_with_parameters(NotPrepared)`:        "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).QueryNonQuery`:                                           "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Same_command_different_param_instances`:                  "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Same_command_different_param_values`:                     "flaky",
@@ -691,6 +693,7 @@ var npgsqlIgnoreList = blocklist{
 	`Npgsql.Tests.CommandTests(NonMultiplexing).Cursor_move_RecordsAffected `:                         "flaky",
 	`Npgsql.Tests.CommandTests(NonMultiplexing).Statement_mapped_output_parameters(SequentialAccess)`: "flaky",
 	`Npgsql.Tests.CommandTests(NonMultiplexing).Use_across_connection_change(Prepared)`:               "flaky",
+	`Npgsql.Tests.ConnectionTests(Multiplexing).Fail_connect_then_succeed(True)`:                      "flaky",
 	`Npgsql.Tests.ConnectionTests(NonMultiplexing).PostgreSqlVersion_ServerVersion`:                   "flaky",
 	`Npgsql.Tests.ConnectionTests(NonMultiplexing).Connector_not_initialized_exception`:               "flaky",
 	`Npgsql.Tests.ConnectionTests(NonMultiplexing).Many_open_close_with_transaction`:                  "flaky",
@@ -730,6 +733,7 @@ var npgsqlIgnoreList = blocklist{
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Wrong_format_binary_export`:                              "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Wrong_format_binary_import`:                              "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Wrong_format_raw_binary_copy`:                            "flaky",
+	`Npgsql.Tests.CopyTests(NonMultiplexing).Wrong_format_text_import`:                                "flaky",
 	`Npgsql.Tests.NotificationTests.WaitAsync_with_timeout`:                                           "flaky",
 	`Npgsql.Tests.NotificationTests.Wait_with_timeout`:                                                "flaky",
 	`Npgsql.Tests.ReaderTests(Multiplexing,Default).Cleans_up_ok_with_dispose_calls(NotPrepared)`:     "flaky",

